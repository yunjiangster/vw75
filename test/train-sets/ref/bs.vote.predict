0.000000 0.000000 0.000000
0.000000 0.000000 0.129255
0.000000 0.000000 0.148772
0.000000 0.000000 0.060840
0.000000 0.000000 0.058650
0.000000 0.000000 0.081537
-1.000000 0.054090 0.092738
-1.000000 0.153191 0.197304
-1.000000 0.070324 0.119705
-1.000000 0.101056 0.156953
-1.000000 0.130891 0.180822
-1.000000 0.117294 0.255996
-1.000000 0.054422 0.219087
-1.000000 0.124910 0.269235
-1.000000 0.091279 0.181126
-1.000000 0.188756 0.303769
-1.000000 0.159724 0.252746
-1.000000 0.212422 0.238775
-1.000000 0.343662 0.438461
-1.000000 0.123063 0.214849
-1.000000 0.080814 0.145011
-1.000000 0.218482 0.427547
-1.000000 0.187381 0.354560
-1.000000 0.166374 0.240115
-1.000000 0.111682 0.133645
-1.000000 0.233228 0.349332
-1.000000 0.132481 0.220394
-1.000000 0.187226 0.301187
-1.000000 0.221044 0.334170
-1.000000 0.188817 0.317579
-1.000000 0.284723 0.468423
-1.000000 0.143435 0.218805
-1.000000 0.127133 0.194157
-1.000000 0.261061 0.329021
-1.000000 0.279921 0.432655
-1.000000 0.232706 0.456617
-1.000000 0.220596 0.374083
-1.000000 0.213011 0.391278
-1.000000 0.311059 0.385609
-1.000000 0.373354 0.430018
-1.000000 0.179937 0.271994
-1.000000 0.247526 0.395931
-1.000000 0.282071 0.401027
-1.000000 0.211439 0.310957
-1.000000 0.242959 0.304165
-1.000000 0.178359 0.250527
-1.000000 0.288963 0.511816
-1.000000 0.275381 0.424339
-1.000000 0.104567 0.276560
-1.000000 0.156771 0.201484
-1.000000 0.217556 0.294967
-1.000000 0.264531 0.299443
-1.000000 0.344519 0.361431
-1.000000 0.289968 0.367334
-1.000000 0.350157 0.410852
-1.000000 0.475434 0.618154
-1.000000 0.371860 0.457569
-1.000000 0.281222 0.306811
-1.000000 0.402672 0.441697
-1.000000 0.294134 0.351652
-1.000000 0.275486 0.287581
-1.000000 0.367807 0.428396
-1.000000 0.268954 0.343244
-1.000000 0.183070 0.334863
-1.000000 0.354704 0.470403
-1.000000 0.223401 0.321164
-1.000000 0.271091 0.315755
-1.000000 0.505197 0.656213
-1.000000 0.278541 0.373263
-1.000000 0.320570 0.502509
-1.000000 0.335556 0.389086
-1.000000 0.336352 0.426043
-1.000000 0.186206 0.359328
-1.000000 0.249785 0.356628
-1.000000 0.212496 0.435216
-1.000000 0.261468 0.314170
-1.000000 0.189441 0.406245
-1.000000 0.324027 0.520712
-1.000000 0.141117 0.163067
-1.000000 0.145962 0.227358
-1.000000 0.214830 0.236126
-1.000000 0.222470 0.332276
-1.000000 0.433449 0.525834
-1.000000 0.233370 0.347997
-1.000000 0.215155 0.405782
-1.000000 0.214738 0.289751
-1.000000 0.218478 0.245169
-1.000000 0.299561 0.566262
-1.000000 0.468340 0.572992
-1.000000 0.212349 0.385987
-1.000000 0.110848 0.166979
-1.000000 0.233616 0.422642
-1.000000 0.633297 0.826363
-1.000000 0.145642 0.230299
-1.000000 0.377151 0.436075
-1.000000 0.259501 0.401401
-1.000000 0.412587 0.511769
-1.000000 0.198737 0.233953
-1.000000 0.317354 0.401689
-1.000000 0.244351 0.331920
-1.000000 0.480473 0.717856
-1.000000 0.403211 0.501961
-1.000000 0.303959 0.434508
-1.000000 0.373397 0.450278
-1.000000 0.348406 0.653601
-1.000000 0.432462 0.524369
-1.000000 0.142839 0.218150
-1.000000 0.307718 0.420620
-1.000000 0.171755 0.274056
-1.000000 0.325026 0.606870
-1.000000 0.481954 0.594186
-1.000000 0.299529 0.583124
-1.000000 0.377000 0.468325
-1.000000 0.227924 0.309569
-1.000000 0.301284 0.378719
-1.000000 0.378061 0.605803
-1.000000 0.285713 0.333442
-1.000000 0.405378 0.530091
-1.000000 0.233560 0.548537
-1.000000 0.342267 0.447244
-1.000000 0.251989 0.394234
-1.000000 0.421192 0.486798
-1.000000 0.384190 0.454336
-1.000000 0.350609 0.487498
-1.000000 0.487036 0.596241
-1.000000 0.271770 0.366433
-1.000000 0.311489 0.400618
-1.000000 0.420472 0.612422
-1.000000 0.151113 0.359416
-1.000000 0.257073 0.447229
-1.000000 0.287476 0.385327
-1.000000 0.474366 0.747045
-1.000000 0.208983 0.240258
-1.000000 0.274090 0.326845
-1.000000 0.361627 0.419766
-1.000000 0.202338 0.337063
-1.000000 0.499522 0.550079
-1.000000 0.350736 0.470842
-1.000000 0.561508 0.677650
-1.000000 0.316386 0.430183
-1.000000 0.390298 0.490941
-1.000000 0.220243 0.274098
-1.000000 0.547527 0.635406
-1.000000 0.651721 0.721642
-1.000000 0.408193 0.532291
-1.000000 0.398598 0.527048
-1.000000 0.400902 0.514689
-1.000000 0.767663 0.851948
-1.000000 0.282213 0.470593
-1.000000 0.316453 0.432482
-1.000000 0.304319 0.657370
-1.000000 0.264833 0.376101
-1.000000 0.497231 0.645729
-1.000000 0.445375 0.536909
-1.000000 0.557215 0.792526
-1.000000 0.468863 0.596351
-1.000000 0.694411 0.816722
-1.000000 0.481922 0.579021
-1.000000 0.585024 0.737615
-1.000000 0.430900 0.556971
-1.000000 0.419828 0.669749
-1.000000 0.265618 0.448421
-1.000000 0.321363 0.608386
-1.000000 0.637512 0.720259
-1.000000 0.311826 0.346903
-1.000000 0.358333 0.459410
-1.000000 0.688031 0.853762
-1.000000 0.343303 0.520371
-1.000000 0.483164 0.533473
-1.000000 0.418838 0.851699
-1.000000 0.567937 0.681369
-1.000000 0.332325 0.413003
-1.000000 0.235746 0.326395
-1.000000 0.335297 0.366862
-1.000000 0.234184 0.269861
-1.000000 0.522422 0.604063
-1.000000 0.302371 0.367260
-1.000000 0.707487 0.874361
-1.000000 0.519989 0.668783
-1.000000 0.679398 0.759810
-1.000000 0.442201 0.549507
-1.000000 0.380661 0.479896
-1.000000 0.424426 0.527809
-1.000000 0.333599 0.434412
-1.000000 0.272487 0.462862
-1.000000 0.353923 0.573783
-1.000000 0.565618 0.794532
-1.000000 0.543055 0.695957
-1.000000 0.254276 0.379602
-1.000000 0.545354 0.694326
-1.000000 0.469327 0.658977
-1.000000 0.211325 0.347316
-1.000000 0.209380 0.298325
-1.000000 0.281265 0.308287
-1.000000 0.323577 0.437014
-1.000000 0.405163 0.642844
-1.000000 0.425380 0.539124
-1.000000 0.287275 0.446434
-1.000000 0.298307 0.469021
-1.000000 0.485682 0.650517
