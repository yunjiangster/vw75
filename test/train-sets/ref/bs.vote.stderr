final_regressor = models/bs.vote.model
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
predictions = bs.vote.predict
using no cache
Reading datafile = train-sets/0001.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
1.000000   1.000000            1         1.0   1.0000   0.0000       51
0.500000   0.000000            2         2.0   0.0000   0.0000      104
0.250000   0.000000            4         4.0   0.0000   0.0000      135
0.500000   0.750000            8         8.0   0.0000  -1.0000      146
0.750000   1.000000           16        16.0   1.0000  -1.0000       24
0.875000   1.000000           32        32.0   0.0000  -1.0000       32
0.937500   1.000000           64        64.0   0.0000  -1.0000       61
0.968750   1.000000          128       128.0   1.0000  -1.0000      106

finished run
number of examples per pass = 200
passes used = 1
weighted example sum = 200
weighted label sum = 0
average loss = 0.98
best constant = 0
total feature number = 15482
